doc-content/enterprise-only/openshift/environment-authoring-con.adoc

```diff
@@ -1,16 +1,46 @@
+For {PRODUCT}, the procedures to deploy an authoring environment and a managed server environment are the same. You must first deploy an authoring environment template, consisting of {CENTRAL} and one {KIE_SERVER}. Then, if necessary, you can deploy additional {KIE_SERVER} templates in the same namespace to create an environment  with multiple {KIE_SERVERS}.
+endif::DM[]
 
-You can also deploy additional managed or immutable {KIE_SERVERS}, if required. {CENTRAL} can automatically discover any {KIE_SERVERS} in the same namespace, including immutable {KIE_SERVERS} and managed {KIE_SERVERS}. This feature requires the `OpenShiftStartupStrategy` setting, which is enabled for all {KIE_SERVERS} except those deployed in a fixed managed infrastructure. For instructions about deploying managed {KIE_SERVERS} with the `OpenShiftStartupStrategy` setting enabled, see {URL_DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT}[_{DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT}_]. For instructions about deploying immutable {KIE_SERVERS}, see {URL_DEPLOYING_IMMUTABLE_ON_OPENSHIFT}[_{DEPLOYING_IMMUTABLE_ON_OPENSHIFT}_].
+Depending on your needs, you can deploy either a single authoring environment template or a high-availability (HA) authoring environment template.
 
-[IMPORTANT]
-====
-In {PRODUCT} {PRODUCT_VERSION}, high-availability {CENTRAL} functionality is for Technology Preview only. For more information on Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope].
-====
+A single authoring environment contains two pods. One of the pods runs {CENTRAL}, the other runs {KIE_SERVER}. 
+ifdef::PAM[The {KIE_SERVER} by default includes an embedded H2 database engine.]
+This environment is most suitable for single-user authoring or when your OpenShift infrastructure has limited resources. 
+
+In an HA authoring environment, both {CENTRAL} and {KIE_SERVER} are provided in scalable pods. When pods are scaled, persistent storage is shared between the copies. 
+ifdef::PAM[The database is provided by a separate pod.]
+To enable high-availability functionality in {CENTRAL}, additional pods with AMQ and Data Grid are required. These pods are configured and deployed by the high-availability authoring template. Use a high-availability authoring environment to provide maximum reliability and responsiveness, especially if several users are involved in authoring at the same time.
+
+In the current version of {PRODUCT}, an HA authoring environment is supported with certain limitations:
+
+* If one user is deploying a service to the {KIE_SERVER}, another user will not be able to deploy a service to the same {KIE_SERVER} until the first deployment completes. 
```

Comment:
Could be add there steps or link to somewhere (maybe RH solution) how to change it to the old startup strategy so user can make changes if they needed.

Reply:
Documenting a template change with adding about 15 lines, as this case it, to production doc standards is not an easy task. Especially because the templates themselves are already finalized. 

What I would want to do is just put in the additions into the template but comment them out. and tell the user in the doc to uncomment the changes. Or at least place a comment where the lines need to be added, and tell the user to find the comment and add the lines.

None of this seems viable, so how would you want me to do it?

Reply:
Maybe the best for now will be to leave the doc as it is and if someone ask just refer them to RHDM-1151. After we have fix for this issue we can provide documentation how to set it properly.

Reply:
Good, leaving as is, just modifying the limitation to show it does not just apply to HA.